docs/about/whats_new.rst

```diff
@@ -8,6 +8,12 @@ What's New
 v1.8.next
 =========
 
+v1.8.15 (11th July 2023)
+========================
+- Replace `entry_points(group=group)` with `entry_points().select(group=group)` for python <3.10 compatibility
```

Comment:
I'm actually not using the `.select()` method, I'm importing `importlib_metadata` for python < 3.10